vendor.conf

```diff
@@ -2,8 +2,8 @@ github.com/BurntSushi/toml master
 github.com/Microsoft/go-winio 307e919c663683a9000576fdc855acaf9534c165
 github.com/Microsoft/hcsshim a8d9cc56cbce765a7eebdf4792e6ceceeff3edb8
 github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76
-github.com/docker/engine-api 4290f40c056686fcaa5c9caf02eac1dde9315adf
 github.com/docker/go-units 0dadbb0345b35ec7ef35e228dabb8de89a65bf52
```

Comment:
should this go-units line be after the go-shellwords line?

Reply:
oh if it's meant to be alphabetically ordered, yes. I can update 😉

Reply:
@TomSweeneyRedHat wait no `docker/go-units` vs `mattn/go-shellwords` :stuck_out_tongue_closed_eyes: